server/images.go

```diff
@@ -235,27 +259,24 @@ func GetModel(name string) (*Model, error) {
 			model.AdapterPaths = append(model.AdapterPaths, filename)
 		case "application/vnd.ollama.image.projector":
 			model.ProjectorPaths = append(model.ProjectorPaths, filename)
-		case "application/vnd.ollama.image.template":
+		case "application/vnd.ollama.image.prompt",
```

Comment:
good catch – how common is this layer type?

Reply:
it shouldn't be very common. it was only implement in the first few iterations of ollama